libs/langchain-openai/src/chat_models.ts

```diff
@@ -1043,28 +1039,24 @@ export class ChatOpenAI<
       modelName: "model",
       openAIApiKey: "openai_api_key",
       apiKey: "openai_api_key",
-      azureOpenAIApiVersion: "azure_openai_api_version",
-      azureOpenAIApiKey: "azure_openai_api_key",
-      azureOpenAIApiInstanceName: "azure_openai_api_instance_name",
-      azureOpenAIApiDeploymentName: "azure_openai_api_deployment_name",
     };
   }
 
-  temperature = 1;
+  temperature?: number;
```

Comment:
just confirming passing undefined = default here, not null (pretty sure this is the case but typescript always makes this fun)

Reply:
I may need to just not specify it when invoking

Reply:
Seems fine